toolkit/tools/boilerplate/boilerplate.go

```diff
@@ -20,13 +21,17 @@ var (
 
 	logFile  = exe.LogFileFlag(app)
 	logLevel = exe.LogLevelFlag(app)
+
+	timestampFile = "boilerplate.csv" // In other go programs, use "app.Flag("timestamp-file", "File that stores timestamps for this program.").Required().String()".
```

Comment:
Why not just use `app.Flags` here as well and make it identical as in other tools? I think there's no need to make people always manually replace this variable whenever we use the boilerplate template for a new tool.